### PR TITLE
Add action for checking groovy syntax

### PR DESCRIPTION
**Pull request by sturivny**

### CODE REVIEW

File: .github/workflows/pipeline_linter.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs check for all the Jenkins files
+      - name: Check JenkinsfileScratchBuildTrigger file
+        run: |
+          sh ./tests/pipeline_linter.sh -t ${{ secrets.OSCI_JENKINS_2_FEDORAPROJECT }} -f JenkinsfileScratchBuildTrigger
```
Comment (bgoncalv) [inline .github/workflows/pipeline_linter.yml:26]: can you try to run the script using `bash` instead of `sh`?

Comment (sturivny) [reply .github/workflows/pipeline_linter.yml]: Yeah, it works!
Thanks =)

Comment (bgoncalv) [reply .github/workflows/pipeline_linter.yml]: cool. Can you squash the commits? Then we can merge it.

Comment (sturivny) [reply .github/workflows/pipeline_linter.yml]: To be honest, I did it from the browser. I don't have access to the `.git` directory from the command line.
Under the `Merge pull request` button there is the drop-down list with `Squash and merge` Could you try it out? :)

**Pull request merged**